packages/eslint-plugin-internal/src/rules/plugin-test-formatting.ts

```diff
@@ -397,7 +402,7 @@ export default createRule<Options, MessageIds>({
               .join('\n')
           : formatted;
 
-        return context.report({
```

Comment:
which rule suggested this?
TBH I prefer the `return report` style here as it's easier to see that the report is terminal.